Automattic/jetpack
# Backup: Copy backup-helper endpoints to backup package

In order for the Backups plugin to fully work without Jetpack, we need to move the backup related endpoints to the `backup` package so that they can be consumed by both plugins independently.

#### Changes proposed in this Pull Request:

*  Backup Endpoints migrated to v4 REST API

#### Jetpack product discussion


1199658893173824-as-1200402244140846/f

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:
- Create a JN site from this PR and make sure it's fully connected (site connected and admin user connected)
- Follow the instructions here: 29934-pb/#php
